resources/building-mobile-extensions/building-an-extension.md

```diff
@@ -47,7 +47,7 @@ The `Extension` class has the following methods that you can optionally override
 
 * `getVersion`, which returns a version string for your extension.  The version string is only used for logging and is currently not validated for formatting.
 * `onUnregistered`, which allows your extension to complete the cleanup that is required when the Adobe Experience Platform SDK unregisters your extension. Unregistration typically happens when you shutdown the app, but it can also occur when an extension is behaving badly. Examples of the extension behaving badly include taking too long to handle a callback or throwing an exception.
-* `onUnexpectedError`, which allows you log additional information when the Adobe Experience Cloud Platform SDK encounters an error that could not be returned immediately from a call into the Adobe Experience Platform SDK.   An example is an exception that is thrown on a worker thread. The exceptions are rare after your extension has been correctly implemented, but the exceptions might occur during development.
+* `onUnexpectedError`, which allows you log additional information when the Adobe Experience Platform SDK encounters an error that could not be returned immediately from a call into the Adobe Experience Platform SDK.   An example is an exception that is thrown on a worker thread. The exceptions are rare after your extension has been correctly implemented, but the exceptions might occur during development.
```

Comment:
- Please add "to" between "you" and "log".
- Seems to be a couple of extra spaces between "...Platform SDK." and "An".
- Please reword the 1st 2 sentences, so that they read as follows:

"`onUnexpectedError` allows you to log additional information when the Adobe Experience Platform SDK encounters an error that could not be immediately returned from a call to the Adobe Experience Platform SDK. An example of this error is an exception that is thrown on a worker thread."